packages/flutter_test/lib/src/mock_canvas.dart

```diff
@@ -6,9 +6,13 @@ import 'dart:ui' as ui show Paragraph, Image;
 
```

Comment:
It would be very helpful to add some code examples to the docs to showcase how to use the APIs.